Jenkinsfile

```diff
@@ -31,8 +31,10 @@ void job_step_update(def value=currentBuild.currentResult) {
     jobStatusUpdate(job_status_internal, env.STAGE_NAME, value)
 }
 
-// For master, this is just some wildly high number
-next_version = '1000'
+// Should try to figure this out automatically
+/* groovylint-disable-next-line CompileStatic, VariableName */
+String base_branch = 'release/2.6'
+String next_version = base_branch
```

Comment:
This should get converted to a number via https://github.com/daos-stack/trusted-pipeline-lib/blob/master/vars/daosLatestVersion.groovy#L49.  We probably should do something similar in the master versions of these branches to make this update easier in the future.

Reply:
There's also this logic
https://github.com/daos-stack/trusted-pipeline-lib/blob/68a64a8fd6b7045b1dc73c1abff577897cae0918/vars/daosLatestVersion.groovy#L36-L37
Maybe it's simpler to just set this to null?

Reply:
It seems the logic you pointed to converts `release/2.6` to `2.6`. How is 2.6 the "next" version?

Reply:
> It seems the logic you pointed to converts `release/2.6` to `2.6`. How is 2.6 the "next" version?

It doesn't.  It converts it to 2.7: https://github.com/daos-stack/trusted-pipeline-lib/blob/master/vars/daosLatestVersion.groovy#L52